### DISCUSSION TITLE
Distance travelled

### DISCUSSION THREAD

**Discussion by BarryCole** [EdgeTX/edgetx#5516] [Ideas]
Sorry if this is in the wrong place, but I cannot find anywhere else.

Is it possible to add a telemetry line to measure "Distance travelled" using GPS speed as the sensor.

Thanks for your help.

**Comment by Helipro** [comment]
> Sorry if this is in the wrong place, but I cannot find anywhere else.
> 
> Is it possible to add a telemetry line to measure "Distance travelled" using GPS speed as the sensor.
> 
> Thanks for your help.

INAV has an odometer and record distant traveled until its reset.

**Reply by BarryCole** [reply]
I am flying model aircraft, not quads, so no INAV.

I would still like, a telemetry line to measure "Distance travelled" using GPS speed as the sensor.

**Comment by ChrisOhara57** [comment]
There is a way to do this with a calculated sensor. 
See the link below (but it is a German forum and the tread is four years old).
I've tested it 3 years ago. It does work.

(With OpenTX 2.3.7  I had to change the units of GSpd to [A] or [mA] and I had to use a Ratio of 1.852 because OpenTx worked with [ktns])

First step try this:
![Screenshot 2024-09-26 194441](https://github.com/user-attachments/assets/97d3d36c-f947-4ff0-a9f5-2c3a71e9a29b)


https://fpv-community.de/threads/wegstrecke-mit-gps-lazy-carbo-way.85758/

**Comment by BarryCole** [comment]
Yes, I have been using that for the last two years. If you change the Gspd to Km/H, the result is in meters, but it calls them mAH.
 I just think it would be nice to have, new line to do it properly.

**Reply by ChrisOhara57** [reply]
If the units are wrong then I think you should place this as a bug.
Your descriptions let's the devs think you want something  new.
I think you should change your request to get the correct units when you use consumption with something else than [A].
